### DISCUSSION TITLE
implement SNe likelihoods for the SRD

### DISCUSSION THREAD

**Discussion by beckermr** [LSSTDESC/firecrown#599] [Ideas]
^ what it says up there

**Comment by slosar** [comment]
As of Feb 2020. This ticket should constist of:
 * implementing a SN likelihood format in sacc -- potentially a separate issue in the sacc repo
 * implementing appropriate reader and model in firecrown.

This ticket should be assigned to renee hlozek.

**Comment by beckermr** [comment]
@reneehlozek

**Comment by marcpaterno** [comment]
@am610 is this something that you are currently working on? If so, can you provide any update on the issue?

**Comment by am610** [comment]
@marcpaterno at the moment, progress is stalled, because of more attention on Firecrown integration in TD. But I will be resuming work on this soon (~ in about 2 weeks)

**Comment by marcpaterno** [comment]
@djbrout can you tell us whether the SNe example in `firecrown/examples/srd_sn` is sufficient to close this (now very old) issue?
